app/src/components/main/tooltips/hoverableTooltip/withHoverableTooltip.jsx

```diff
@@ -16,16 +16,38 @@ export const withHoverableTooltip = ({ TooltipComponent, data }) => WrappedCompo
     };
     componentDidMount() {
       const tooltipWidth = data.width || DEFAULT_TOOLTIP_WIDTH;
+      let left = 0;
       this.tooltip.style.width = `${tooltipWidth}px`;
-      this.tooltip.style.left = `${(this.hoverRect.offsetWidth / 2) - (tooltipWidth / 2)}px`;
+      switch (data.align) {
+        case 'left':
```

Comment:
'left' and 'right' should be constants for both tooltip types

Reply:
fixed